README.rst

```diff
@@ -24,184 +20,15 @@ data. It is the spiritual successor to the original *kafe* package.
-dependencies. Please note that commands below should be run as root.
-
-For Python 2:
-
-.. code:: bash
-
-    pip2 install kafe2
-
-
-
-For Python 3:
-
-.. code:: bash
-
-    pip3 install kafe2
-
-
-If you don't have *pip* installed, get it from the package manager.
-
-In Ubuntu/Mint/Debian, do:
-
-.. code:: bash
-
-    apt-get install python-pip python3-pip
-
-
-In Fedora/RHEL/CentOS, do:
-
-.. code:: bash
-
-    yum install python2-pip python3-pip
-
-
-or use ``easy_install`` (included with `setuptools <https://pypi.python.org/pypi/setuptools>`_):
-
-.. code:: bash
-
-    easy_install pip
-
-
-You will also need to install *Tkinter* if it didn't already come with your Python distribution.
-
-For Python 2, Ubuntu/Mint/Debian:
-
-.. code:: bash
-
-    apt-get install python-tk
-
-
-For Python 2, Fedora/RHEL/CentOS:
-
-.. code:: bash
-
-    yum install tkinter
-
-
-For Python 3, Ubuntu/Mint/Debian:
-
-.. code:: bash
-
-    apt-get install python3-tk
-
-
-For Python 3, Fedora/RHEL/CentOS:
-
-.. code:: bash
-
-    yum install python3-tkinter
-
-
-------------------------
-Optional: Install *ROOT*
-------------------------
-
-**Note: Starting with Ubuntu 16.10, ROOT is no longer available in the official repositories.**
-
-In older versions of Ubuntu (and related Linux distributions), ROOT and its Python bindings
-can be obtained via the package manager via:
-
-.. code:: bash
-
-    apt-get install root-system libroot-bindings-python5.34 libroot-bindings-python-dev
-
-
-Or, in Fedora/RHEL/CentOS:
-
-.. code:: bash
-
-    yum install root root-python
-
-
-This setup is usually sufficient. However, you may decide to build ROOT yourself. In this case,
-be sure to compile with *PyROOT* support. Additionally, for Python to see the *PyROOT* bindings,
-the following environment variables have to be set correctly:
-
-.. code:: bash
-
-    export ROOTSYS=<directory where ROOT is installed>
-    export LD_LIBRARY_PATH=$ROOTSYS/lib:$PYTHONDIR/lib:$LD_LIBRARY_PATH
-    export PYTHONPATH=$ROOTSYS/lib:$PYTHONPATH
-
-
-For more info, refer to `<http://root.cern.ch/drupal/content/pyroot>`_.
-
-
----------------------------
-Optional: Install `iminuit`
----------------------------
-
-*iminuit* is a Python wrapper for the Minuit minimizer which is
-independent of ROOT. This minimizer can be used instead of ROOT.
-
-To install the *iminuit* package for Python, the `Pip installer
-<http://www.pip-installer.org/>`_ is recommended:
-
-.. code:: bash
-
-    pip install iminuit
-
-You might also need to install the Python headers for *iminuit* to
-compile properly.
-
-In Ubuntu/Mint/Debian, do:
-
-.. code:: bash
-
-    apt-get install libpython2-dev libpython3-dev
-
-In Fedora/RHEL/CentOS, do:
-
-.. code:: bash
-
-    yum install python2-devel python3-devel
-
+A user guide, including installation instructions, can be found under `kafe2.readthedocs.io <https://kafe2.readthedocs.io/en/latest/parts/user_guide.html>`_.
```

Comment:
The installation instructions are located at [https://kafe2.readthedocs.io/en/latest/parts/installation.html](https://kafe2.readthedocs.io/en/latest/parts/installation.html).

I plan to refactor the current "User Guide" to "Beginners Guide" and create a "User Guide" with more specific information on how to do certain things. E.g. enable logging, customize the plots, using the ContoursProfiler etc.